test/filter-test.js

```diff
@@ -41,7 +40,7 @@ describe('Filter', function() {
   function write(relativePath, contents, _encoding) {
     let encoding = _encoding === undefined ? 'utf8' : _encoding;
 
-    mkdirp.sync(path.dirname(relativePath));
+    fs.mkdirSync(path.dirname(relativePath), { recursive: true });
```

Comment:
FYI - I actually don't think this is allowed in our current engines range. We specify support for 10.0.0 (via `10.*` in engines), but this API was not added until 10.12.0.

Since this is only used in tests, I think it doesn't really matter too much.

Reply:
That's what I thought, 10.12 came out almost 2 years ago so hopefully something only used in tests wouldn't need to work that far back.